.meta/sinks/humio_logs.toml.erb

```diff
@@ -95,4 +95,16 @@ description = "The optional endpoint to send Humio logs to."
   namespace: "sinks.humio_logs.options",
   encodings: ["json", "text"],
   default: "json"
-) %>
+          ) %>
+
+[sinks.humio_logs.options.host_key]
+type = "string"
+common = true
+examples = ["hostname"]
+required = false
+description = """\
+The name of the log field to be used as the hostname sent to Humio. This \
+overrides the \ [global `host_key` \
```

Comment:
Extra \ here perhaps? I see the check-meta failing.

Reply:
Oops, thanks! I line wrapped and miss that `\`.